src/screencast_keys/ops.py

```diff
@@ -1507,3 +1539,23 @@ def invoke(self, context, event):
         self.draw_handler_add(context)
         context.window_manager.modal_handler_add(self)
         return {'RUNNING_MODAL'}
+
+
+@BlClassRegistry()
+class SK_OT_WaitForInitialize(bpy.types.Operator):
```

Comment:
I think `SK_OT_WaitBlenderInitializedAndStartScreencastKeys` is better name to understand this operator.

Reply:
Yes